ide/provisioning/install-debian-packages.sh

```
@@ -116,7 +116,8 @@ install_arm() {
 
 install_win() {
   echo Installing PC/WIN64 dependencies...
-  apt-get install ${APTOPTS[*]} g++-mingw-w64
+  apt-get install ${APTOPTS[*]} g++-mingw-w64 \
```

Shellcheck (0.11.0)

[warning] Use "${array[@]}" (with quotes) to prevent whitespace problems.

(SC2048)